src/main/java/com/cs525/reversi/models/GameBuilder.java

```diff
@@ -0,0 +1,17 @@
+package com.cs525.reversi.models;
+
+public interface GameBuilder {
+
+	public GameBuilder buildPlayerOne(User playerOne);
+
+	public GameBuilder buildPlayerTwo(User playerTwo);
+
+	public ReversiGameBuilder buildGameStatus(GameStatus gameStatus);
```

Comment:
@MustafaMosad Is there a reason for using the concrete `ReversiGameBuilder` instead of `GameBuilder` here and the 2 methods below?